io.github.trigg.discover_overlay.yaml

```
@@ -0,0 +1,16 @@
+app-id: io.github.trigg.discover_overlay
+runtime: org.gnome.Platform
+runtime-version: '3.38'
```

```suggestion
runtime-version: '41'
```

this version is deprecated

---

also does it really need the GNOME platform?

---

It doesn't work with Freedesktop or KDE platforms.